server/views/applications/new.njk

```diff
@@ -22,7 +22,7 @@
           {% set html %}
           <p class='govuk-body'>You do not have access to this offender as this person is a limited access offender (LAO).</p>
           <p class='govuk-body'>You cannot continue with the application without requesting LAO access.</p>
-          <p class='govuk-body'>Please follow the guidance in Equip to remove the restriction if this is not correct or to request access.</p>
+          <p class='govuk-body'>Please follow <a href="https://equip-portal.equip.service.justice.gov.uk/CtrlWebIsapi.dll/?__id=webDiagram.show&map=0%3AFF2D8D3F16B44268B814F7F8177A16F7&dgm=0CCF9118DFAC43F697E66CE331889D60">the guidance in EQuiP</a> to remove the restriction if this is not correct or to request access.</p>
```

Comment:
I wonder if a `target="_blank"` on the `a` would be useful here so it opens in a new tab?